cocoapods-plugin/lib/cocoapods-xcremotecache/command/hooks.rb

```diff
@@ -338,7 +341,6 @@ def self.save_lldbinit_rewrite(user_proj_directory)
           end
 
           validate_configuration()
-          disable_certificate_verification = @@configuration['disable_certificate_verification']
```

Comment:
unused

Reply:
why did you remove it from here?

Reply:
Don't you agree that it is unused? Maybe differently: why do you think it is required?

Reply:
it is currently not used but will be used in the future. isn't it populated here from the configuration that is parsed from the podfile?

Reply:
Yes, this local variable is fetched from the podfile and just dangling. Keeping a local variable if it is not used is not a common pattern and many linters just mark it as a rules violation.
I don't see a reason why it might be used in the future here. The value `true`/`false` from the Podfile is moved to the `.rcinfo` anyway thanks to `CUSTOM_CONFIGURATION_KEYS`.

Reply:
well. Thank you!